Doc/library/os.path.rst

```diff
@@ -306,11 +306,13 @@ the :mod:`glob` module.)
 
    Join one or more path components intelligently.  The return value is the
    concatenation of *path* and any members of *\*paths* with exactly one
-   directory separator (``os.sep``) following each non-empty part except the
-   last, meaning that the result will only end in a separator if the last
-   part is empty.  If a component is an absolute path, all previous
-   components are thrown away and joining continues from the absolute path
-   component.
+   directory separator following each non-empty part except the last, meaning
```

Comment:
Since the actual change here is so small and clearly a removal, I'd recommend not re-wrapping the paragraph.  The resulting patch would be smaller and easier to understand.

Does't require changing here, but something to keep in mind in the future.